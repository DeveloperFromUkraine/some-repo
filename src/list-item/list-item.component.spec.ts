import { ComponentFixture, TestBed } from "@angular/core/testing";
import { ListItemComponent } from "../index";
import { ComponentTest } from "../../test/test-bed/component";

describe('List Item', () => {
    let fixture: ComponentFixture<ListItemComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [ListItemComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(ListItemComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});