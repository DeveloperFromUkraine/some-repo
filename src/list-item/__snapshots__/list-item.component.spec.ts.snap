// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Item should match snapshot 1`] = `
<ign-list-item>
  
</ign-list-item>
`;
